packages/svelte/src/lib/component/Dropzone.svelte

```diff
@@ -0,0 +1,364 @@
+<script lang="ts">
```

Comment:
can you refactor this to use our `@uploadthing/dropzone/core` ?

Reply:
Sure!